openpathsampling/tools.py

```diff
@@ -205,24 +205,20 @@ def units_from_snapshot(snapshot):
         the simtk.unit.Unit to be used
     """
 
-    units = {}
+    units = {'velocity': None, 'energy': None, 'length': None}
```

Comment:
This is still in there because I split the PR. In the next steps this should be removed and only be used for MDSnapshots. 

I still need this in here because the current implementation uses this to determine that toy snapshots do not have a unit an MD snapshots have. 

This is one of the reasons why the PR became so big. I realized to get rid of this little unit thing I need to implement support for different Snapshot Classes and hence the possibility to store stores(that know which Snapshot they store), etc... one lead to the other.